<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Denoiser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(143, 39, 233);
font-family: &quot;Bernard MT Condensed&quot;;
font: 30pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Audio Denoiser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>400</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/image/mic_img.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QPushButton" name="denoise_wav_file">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(143, 39, 233);
color:rgb(255, 255, 255);
border-width:1px;
border-radius:10px;
border-color: blue;</string>
    </property>
    <property name="text">
     <string>Denoise wav file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="denoise_recording">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(143, 39, 233);
color:rgb(255, 255, 255);
border-width:1px;
border-radius:10px;
border-color: blue;</string>
    </property>
    <property name="text">
     <string>Denoise Live Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(143, 39, 233);
color:rgb(255, 255, 255);
border-width:1px;
border-radius:10px;
border-color: blue;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="ui_resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
